kube.mk

```diff
@@ -40,25 +40,19 @@ generate-server-rbacs: bin/controller-gen
 		rbac:roleName=server-role
 	prettier --write ./config/server-rbac/
 
-REGISTRY ?= quay.io/tinkerbell
-TINK_SERVER_IMAGE_NAME ?= tink-server
-TINK_CONTROLLER_IMAGE_NAME ?= tink-controller
-TINK_SERVER_IMAGE_TAG ?= latest
-TINK_CONTROLLER_IMAGE_TAG ?= latest
-
-out/release/default/manager_image.yaml: config/default/manager_image.yaml out/release/default/kustomization.yaml
-out/release/default/manager_image.yaml: TAG=$(REGISTRY)/$(TINK_CONTROLLER_IMAGE_NAME):$(TINK_CONTROLLER_IMAGE_TAG)
-out/release/default/server_image.yaml: config/default/server_image.yaml out/release/default/kustomization.yaml
-out/release/default/server_image.yaml: TAG=$(REGISTRY)/$(TINK_SERVER_IMAGE_NAME):$(TINK_SERVER_IMAGE_TAG)
-out/release/default/manager_image.yaml out/release/default/server_image.yaml:
-	sed -e 's|image: .*|image: "$(TAG)"|' $^ >$@
+TINK_SERVER_IMAGE ?= quay.io/tinkerbell/tink-server
+TINK_CONTROLLER_IMAGE ?= quay.io/tinkerbell/tink-controller
+TINK_SERVER_TAG ?= latest
+TINK_CONTROLLER_TAG ?= latest
 
 out/release/default/kustomization.yaml: config/default/kustomization.yaml
 	rm -rf out/
 	mkdir -p out/
 	cp -a config/ out/release/
+	sed -e 's|TINK_CONTROLLER_IMAGE|$(TINK_CONTROLLER_IMAGE)|g' -e 's|TINK_CONTROLLER_TAG|$(TINK_CONTROLLER_TAG)|g' \
```

Comment:
kustomize does have an edit command. might be something to consider. for example:

```bash
./bin/kustomize edit set image tink-controller=quay.io/tinkerbell/tink-controller:v0.1.1
```

Reply:
done